x-pack/filebeat/input/awscloudwatch/cloudwatch.go

```diff
@@ -73,36 +73,30 @@ func (p *cloudwatchPoller) run(svc cloudwatchlogsiface.ClientAPI, logGroup strin
 }
 
 // getLogEventsFromCloudWatch uses FilterLogEvents API to collect logs from CloudWatch
-func (p *cloudwatchPoller) getLogEventsFromCloudWatch(svc cloudwatchlogsiface.ClientAPI, logGroup string, startTime int64, endTime int64, logProcessor *logProcessor) error {
+func (p *cloudwatchPoller) getLogEventsFromCloudWatch(svc *cloudwatchlogs.Client, logGroup string, startTime int64, endTime int64, logProcessor *logProcessor) error {
 	// construct FilterLogEventsInput
 	filterLogEventsInput := p.constructFilterLogEventsInput(startTime, endTime, logGroup)
 
 	// make API request
-	req := svc.FilterLogEventsRequest(filterLogEventsInput)
-	paginator := cloudwatchlogs.NewFilterLogEventsPaginator(req)
-	for paginator.Next(context.TODO()) {
```

Comment:
Weird that the paginator has been removed. Note to myself: Double check this @kaiyan-sheng mentions that they added the paginator after a aws library change (so now they remove it again)